lib/LanguageServerRename/LanguageServerRenameWorseExtension.php

```diff
@@ -43,6 +39,19 @@ public function load(ContainerBuilder $container): void
         }, [
             LanguageServerRenameExtension::TAG_RENAMER => []
         ]);
+
+        $container->register(MemberRenamer::class, function (Container $container) {
+            return new MemberRenamer(
+                new DefinitionAndReferenceFinder(
```

Comment:
this class combines the definition finder with the reference finder, so that it includes the definition in the references

Reply:
And now it's not necessary as the variable reference finder will also find the definition (when passed a flag in the constructor)